src/modules/70-pipeline/components/PipelineStudio/SavePipelinePopover/SavePipelinePopover.tsx

```diff
@@ -160,8 +169,11 @@ function SavePipelinePopover(
   })
 
   const isTemplatesEnabled = templatesFeatureEnabled && canEdit && !pipeline?.template
-
-  const isSaveDisabled = isReadonly || !isUpdated || isIntermittentLoading
+  let isSaveDisabled = isReadonly || !isUpdated || isIntermittentLoading || isMetadataUpdated
+  if (isMetadataUpdated) {
```

Comment:
Do you need `|| isMetadataUpdated` here? Doesn't appear like it'll change anything since the next statement flips `isSaveDisabled` if it is true
```suggestion
  let isSaveDisabled = isReadonly || !isUpdated || isIntermittentLoading
  if (isMetadataUpdated) {
```

Reply:
Thanks, got missed while testing